projectile.el

```diff
@@ -1489,36 +1489,45 @@ Never use on many files since it's going to recalculate the
 project-root for every file."
   (expand-file-name name (projectile-project-root)))
 
-(defun projectile-completing-read (prompt choices &optional initial-input)
+(cl-defun projectile-completing-read (prompt choices
```

Comment:
I'm well-aware of the benefits, I just dislike using features not present in Elisp by default (they tend to be confusing to people not familiar with Common Lisp). Anyways, I can live with this.